src/lib/guesstimator/samplers/DistributionBeta.js

```diff
@@ -1,7 +1,7 @@
 import {simulate} from './Simulator.js'
 
 export var Sampler = {
-  sample({params: [hits, total]}, n) {
+  sample({params: [hits, total]}, n, _1) {
```

Comment:
`_1` is just to indicate an unused parameter. `_` is a common convention ime for unused parameters, and javascript doesn't allow it to be used for multiple parameters, but we use that for lodash, so I use `_1`. This is useful because when that function is called, it is actually called with 3 parameters, not two, meaning if you want to add another parameter to it (as I did, then removed, when collected performance data) you need to add an unused placeholder in the third position. Naming that placeholder seems unclear to me as it is unused. 

If you don't like that convention, let me know what you'd like to do instead (either a different convention for an unused parameter, or omit it, or name the parameter despite it being unused, or whatever).